misk/src/main/kotlin/misk/web/MiskWebModule.kt

```diff
@@ -298,6 +300,7 @@ class MiskWebModule @JvmOverloads constructor(
       }
       bind<ThreadPool>().toInstance(threadPool)
       bind<MeasuredThreadPool>().toInstance(MeasuredThreadPoolExecutor(executor))
+      multibind<HealthCheck>().to<JettyThreadPoolHealthCheck>()
```

Comment:
Don't we want this for `QueuedThreadPool` as well?

Reply:
Also, maybe we should make this configurable so we can explicitly opt into it and test it out?

Reply:
yeah adding health checks can be a bit risky as they can cause flapping and sitting ducks, roll it out very carefully

Reply:
Made the health checks configurable and fixed to include `QueuedThreadPool`